src/database/__tests__/testcases.tsx

```diff
@@ -21,7 +21,6 @@ saveFacility(scenarioId, {
     countyName: "Ascension",
     rateOfSpreadFactor: "moderate",
     usePopulationSubsets: true,
-    facilityOccupancyPct: 0.85,
```

Comment:
I don't see anywhere that we're running this test. Should we remove this test file?